contributor-docs/rc-testing-guide.md

```diff
@@ -0,0 +1,39 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Release Candidate (RC) Testing Guide
+
+This guide is meant for anybody who is interested in testing Beam Release Candidates against downstream projects. Note
+that one need not have any status on the Apache Beam project (eg. PMC Member, Committer) to vote; all are welcome.
+Please subscribe to the [dev list](https://lists.apache.org/list.html?dev@beam.apache.org), and vote on the RC Vote email thread.
+
```

Comment:
We should probably have a brief overview of the Beam release process (and especially the voting process), with links to https://github.com/apache/beam/blob/master/contributor-docs/release-guide.md